app/controllers/v0/facilities/va_controller.rb

```diff
@@ -13,7 +13,7 @@ class V0::Facilities::VaController < FacilitiesController
   # @param services - Optional specialty services filter
   def index
     # this first return is temporary for the rollout of provider locator to staging
-    return facilities unless Settings.locators.providers_enabled
+    # return facilities unless Settings.locators.providers_enabled
```

Comment:
I'd prefer to keep this in place until we are ready to release this feature in prod